### PR TITLE
Fix graphene-sys tests

### PR DESCRIPTION
**Pull request by andy128k**

### CODE REVIEW

File: graphene/sys/tests/constant.c [hunk 1]
```diff
@@ -21,6 +21,7 @@
                     unsigned long: "%lu", \
                     long long: "%lld", \
                     unsigned long long: "%llu", \
+                    float: "%f", \
```
Comment (GuillaumeGomez) [inline graphene/sys/tests/constant.c:24]: Why adding this line? Just below there the same printf option.

Comment (andy128k) [reply graphene/sys/tests/constant.c:24]: GCC (v 10.2.1) emits an error:

```
error: ‘_Generic’ selector of type ‘float’ is not compatible with any association
```

Comment (andy128k) [reply graphene/sys/tests/constant.c:24]: Corresponding change in `gir` is [here](https://github.com/gtk-rs/gir/pull/1142).

File: graphene/sys/src/lib.rs [hunk 2]
```diff
@@ -62,9 +62,6 @@ pub const GRAPHENE_RAY_INTERSECTION_KIND_ENTER: graphene_ray_intersection_kind_t
 pub const GRAPHENE_RAY_INTERSECTION_KIND_LEAVE: graphene_ray_intersection_kind_t = 2;
 
 // Constants
-pub const GRAPHENE_HAS_GCC: c_int = 1;
-pub const GRAPHENE_HAS_SCALAR: c_int = 1;
-pub const GRAPHENE_HAS_SSE: c_int = 1;
 pub const GRAPHENE_PI: c_double = 3.141593;
 pub const GRAPHENE_PI_2: c_double = 1.570796;
```
Comment (bilelmoussaoui) [inline graphene/sys/src/lib.rs:66]: I would probably just ignore those & make them an alias of the std types

Comment (sdroege) [reply graphene/sys/src/lib.rs:66]: Yeah, like I wrote in the gir PR :)

File: graphene/sys/Gir.toml [hunk 3]
```diff
@@ -15,6 +15,11 @@ external_libraries = [
 ignore = [
     "Graphene.true",
     "Graphene.false",
+    "Graphene.HAS_GCC",
+    "Graphene.HAS_SCALAR",
+    "Graphene.HAS_SSE",
+    "Graphene.PI",
+    "Graphene.PI_2",
```
Comment (bilelmoussaoui) [inline graphene/sys/Gir.toml:22]: you will have to mark them as manual on the non-sys part & added them to lib.rs

Comment (andy128k) [reply graphene/sys/Gir.toml:22]: Why? These constants are [not documented](https://developer.gnome.org/graphene/stable/api-index.html).

**Comment by bilelmoussaoui** [comment]
Can you re-enable graphene's sys tests on the CI? so we can be sure the PR actually fixes them. Thanks!

**Comment by sdroege** [comment]
If you rebase on latest master then the docs CI should also pass.

File: graphene/sys/src/lib.rs [hunk 4]
```diff
@@ -62,11 +62,6 @@ pub const GRAPHENE_RAY_INTERSECTION_KIND_ENTER: graphene_ray_intersection_kind_t
 pub const GRAPHENE_RAY_INTERSECTION_KIND_LEAVE: graphene_ray_intersection_kind_t = 2;
 
 // Constants
-pub const GRAPHENE_HAS_GCC: c_int = 1;
-pub const GRAPHENE_HAS_SCALAR: c_int = 1;
-pub const GRAPHENE_HAS_SSE: c_int = 1;
-pub const GRAPHENE_PI: c_double = 3.141593;
-pub const GRAPHENE_PI_2: c_double = 1.570796;
 pub const GRAPHENE_SIMD_S: *const c_char = b"sse\0" as *const u8 as *const c_char;
```
Comment (sdroege) [inline graphene/sys/src/lib.rs:65]: This one should also be ignored while we're at it. It's completely dependent on where the graphene version was built that was used to generate the .gir file.

**Review by sdroege — COMMENTED** [review]
Seems good to me otherwise :)

**Review by GuillaumeGomez — APPROVED** [review]

File: .github/workflows/CI.yml [hunk 5]
```diff
@@ -77,6 +77,10 @@ jobs:
       - name: build
         run: cargo build --manifest-path ${{ matrix.conf.name }}/Cargo.toml --features "${{ matrix.conf.features }}"
         if: matrix.rust != 'nightly'
+      - name: Test ${{ matrix.conf.name }}/sys
+        run: cargo test
+        working-directory: ${{ matrix.conf.name }}/sys
+        if: matrix.conf.name == 'graphene'
```
Comment (bilelmoussaoui) [inline .github/workflows/CI.yml:83]: You just need to change the value in https://github.com/gtk-rs/gtk-rs-core/pull/73/files#diff-3ab46ee209a127470fce3c2cf106b1a1dbadbb929a4b5b13656a4bc4ce19c0b8R26

Comment (andy128k) [reply .github/workflows/CI.yml:83]: Parameter `test_sys` does nothing. It is not used anywhere in the `CI` action.

File: graphene/sys/src/lib.rs (hunk 4 above)
Comment (andy128k) [reply graphene/sys/src/lib.rs]: Done

File: .github/workflows/CI.yml (hunk 5 above)
Comment (sdroege) [reply .github/workflows/CI.yml:83]: @bilelmoussaoui What should we do about this?

Comment (bilelmoussaoui) [reply .github/workflows/CI.yml:83]: wait what. It definitely used to do something, any ideas @GuillaumeGomez ?

Comment (bilelmoussaoui) [reply .github/workflows/CI.yml:83]: Please copy it from https://github.com/gtk-rs/gtk4-rs/blob/master/.github/workflows/CI.yml#L37-L39 for now :-)

**Review by bilelmoussaoui — APPROVED** [review]
Thanks!

**Review by GuillaumeGomez — APPROVED** [review]

**Comment by GuillaumeGomez** [comment]
Thanks!

**Pull request merged**